plugin/src/main/java/com/denizenscript/denizen/objects/EntityTag.java

```diff
@@ -1976,14 +1976,14 @@ else if (object.getLivingEntity() instanceof Steerable) {
         // @attribute <EntityTag.is_sleeping>
         // @returns ElementTag(Boolean)
         // @description
-        // Returns whether the player, NPC, or villager is currently sleeping.
+        // Returns whether a living entity is currently sleeping.
+        // Will always return 'false' on entities that are unable to sleep.
+        // This tag may return other values for foxes, players, and villagers.
+        // @mechanism EntityTag.is_sleeping

```

Comment:
Is this saying "This will return `false` for most entities, other than foxes players and villagers which it will work for"? if so that seems like a pretty confusing way to say this imo.
Also just a nitpick but the `@mechanism` link is generally above the description.